test/upload_validation_test.ts

```diff
@@ -771,6 +771,60 @@ describe('Pack metadata Validation', () => {
         ]);
       });
 
+      it('allows a schema object to be reused in 2 sync tables', async () => {
+        const MySchema = makeObjectSchema({
+          properties: {
+            name: {type: ValueType.String},
+            // Add more properties here.
+          },
+          idProperty: 'name',
+          displayProperty: 'name',
+        });
+
+        const table1 = makeSyncTable({
+          name: 'One',
+          description: 'The first sync table.',
+          identityName: 'One',
+          schema: MySchema,
+          formula: {
+            name: 'SyncThings',
+            description: 'Sync some things.',
+            parameters: [],
+            async execute([]) {
+              return {
+                result: [],
+              };
+            },
+          },
+        });
+        const table2 = makeSyncTable({
+          name: 'Two',
+          description: 'The second sync table.',
+          identityName: 'Two',
+          schema: MySchema,
+          formula: {
+            name: 'SyncThings',
+            description: 'Sync some things.',
+            parameters: [],
+            async execute([]) {
+              return {
+                result: [],
+              };
+            },
+          },
+        });
+        const metadata = createFakePack({
+          syncTables: [table1, table2],
+        });
+        const validatedMetadata = await validateJson(metadata);
+
+        const {schema: schema1} = validatedMetadata.syncTables[0];
+        const {schema: schema2} = validatedMetadata.syncTables[1];
+        assert.ok(schema1);
+        assert.ok(schema2);
+        // assert.deepEqual(schema.identity, {name: 'SomeIdentity'});
```

Comment:
There's a commented-out line here. Did you intend to have this assert on the identity of the return values?

Reply:
Thank you for reminding me that I never finished the test!!